frontend/src/pages/login/loginPage.tsx

```diff
@@ -18,23 +18,17 @@ import {
 
 import './login.css'
 import Amplify from '@aws-amplify/core'
-import AuthState from '../../model/authState'
-import { useRegisterUpdate } from '../../model/registerContext'
 
-interface LoginPageProps {
-  setAuthState(authState: AuthState): void
-}
-
-const LoginPage: React.FC<LoginPageProps> = ({ setAuthState }) => {
+const LoginPage: React.FC = (props: any) => {
```

Comment:
you are missing types

Reply:
True, but it's because AWS is not offering any to us....😢